docs/src/code/building-linuxcnc.adoc

```diff
@@ -289,11 +294,38 @@ If you want just the traditional RTAI "kernel module" realtime, use
 == Satisfying Build Dependencies
 
 On Debian-based platforms we provide packaging meta-data that knows
-what external software packages need to be installed in order to build
-LinuxCNC.  This is called the Build Dependencies of LinuxCNC.  You can
-use this meta-data to easily list the required packages missing from
-your build system.
+what external software packages need to be installed in order to build LinuxCNC.
+These are referred to as the _build dependencies_ of LinuxCNC, i.e. those
+packages that need to be available such that
+* the build succeeds and
+* the build can be built reproducibly.
 
+You can use this meta-data to easily list the required packages missing from your build system.
+First, go to the source tree of LinuxCNC and initiate its default self-configuration,
+if not already performed:
+
+-----
+$ cd linuxcnc-dev
+$ ./debian/configure
+-----
+
+This will prepare the file debian/control that contains lists of Debian packages
+to create with the runtime dependencies for those packages and for our cause also
+the build-dependencies for those to-be-created packages.
+
+The most straightforward way to get all build-dependencies installed is to just
+execute (from the same directory):
+
+----
+sudo apt-get build-dep .
```

Comment:
Is it worth stressing that the . is part of the command, not just punctuation?

Reply:
The sentence continues below that verbatim block, but, donno, maybe yes. There could be a caption for that verbatim block with a respective partially redundant explanation of what this apt-get command is doing. I'll come up with something.